activities/MazeWeb.activity/css/activity.css

```diff
@@ -44,4 +44,22 @@
   background-color: #101010;
   overflow-x: hidden;
   overflow-y: hidden;
-}
\ No newline at end of file
+}
+
+#fullscreen-button {
+  background-image: url(../icons/view-fullscreen.svg);
+}
+
+#unfullscreen-button {
+  background-image: url(../icons/view-return.svg);
+  border: 0;
+  border-radius: 0px;
+  margin: 1px;
+  width: 45px;
+  height: 45px;
+  top: 0;
+  left: 95%;
```

Comment:
Should be located from the right side, else there is a side effect on small screens: See https://github.com/llaske/sugarizer/pull/420#issuecomment-552965633